doc/source/tutorial/optimize.rst

```diff
@@ -1342,10 +1342,45 @@ residual is expensive to compute, good preconditioning can be crucial
 --- it can even decide whether the problem is solvable in practice or
 not.
 
+
 Preconditioning is an art, science, and industry. Here, we were lucky
 in making a simple choice that worked reasonably well, but there is a
 lot more depth to this topic than is shown here.
 
+
+Linear Sum Assignment
+^^^^^^^^^^^^^^^^^^^^^^
+
+The linear sum assignment problem is also known as minimum weight matching in
+bipartite graphs. A problem instance is described by a matrix C, where each
+C[i,j] is the cost of matching vertex i of the first partite set (a “worker”)
+and vertex j of the second set. The goal is to find a complete assignment
+of workers for tasks of minimal cost.
+
+Formally, let X be a boolean matrix where :math:`X[i,j] = 1` iff row i is
+assigned to column j. Then the optimal assignment has cost
+
+.. math::
+    \\min \\sum_i \\sum_j C_{i,j} X_{i,j}
+
+s.t. each row is assignment to at most one column, and each column to at most one row.
```

Comment:
Passing by with a quick remark here: This addition duplicates the [docstring for the method as of its 1.3.0 version](https://github.com/scipy/scipy/blob/v1.3.0/scipy/optimize/_hungarian.py#L13-L118). Maybe that's okay? The docstring is accurate enough, but seems like it would be annoying to have to maintain the documentation in two places.

And note that by being based on the old version, it includes a [mistake that was fixed recently](https://github.com/scipy/scipy/pull/10344).